### ISSUE TITLE
Identify Breadcrumbs - Back-end/Devops api

### ISSUE THREAD

**Issue by ZoeSimon** [bcgov/lucy-web#506]
**Describe the Task**
Identify Breadcrumbs for Back end/DevOps api.

Breadcrumbs in this context include: "This exists, it is blocked, here is why, and this is what to do when it is unblocked"

**Comment by ghost** [comment]
Added the list of features for the backend, @fwpushan Can you add the pieces for pipeline?

https://docs.google.com/document/d/1rF9MTJzGIFVv1Ds0yX4TiwAvE9PTJvPscbyxhCWW7-s/edit?usp=sharing

**Comment by ZoeSimon** [comment]
Mike Wells' comment from ticket #495 

- what things are partly done
- what low priority stuff has no tests, but needs doc
- things that were not done, but anticipated to be needed later, and advice on changes that would needed to be made, such as:
- what things would need to change if we moved to high availability i.e. more than one pod running your api.

**Comment by fwpushan** [comment]
@AmirShayegh Please add Breadcrumbs for the Inspect iOS app.
